package.json

```diff
@@ -80,6 +80,7 @@
     "lodash": "^4.17.15",
     "mapbox-gl": "^1.8.1",
     "moment": "^2.24.0",
+    "notistack": "^0.9.17",
```

Comment:
> Is there a reason you strongly prefer this library over [Material-UI Snackbars](https://material-ui.com/components/snackbars/)?
> 
> I'm just asking, because in general I'd like to minimize the number of external dependencies. But if there's a strong reason you prefer this library, it's not a problem!


I added the library since at the bottom of the Material UI Snack bar docs, they recommended using notistack for advanced usecases (stacking notifications), which is really the only big selling point of notistack. If having just one at a time is fine, we could definitely move to just using Material UI. Do you think stacking is worth?